.ibm/pipelines/jobs/aks-helm.sh

```diff
@@ -17,10 +17,10 @@ handle_aks_helm() {
   cluster_setup_k8s_helm
 
   initiate_aks_helm_deployment
-  check_and_test "${RELEASE_NAME}" "${NAME_SPACE}" "https://${K8S_CLUSTER_ROUTER_BASE}" 50 30
+  check_and_test "${RELEASE_NAME}" "${NAME_SPACE}" "showcase-k8s" "https://${K8S_CLUSTER_ROUTER_BASE}" 50 30
```

Comment:
The Playwright project names are now hardcoded across multiple job files. Consider defining them as constants in `env_variables.sh`:
```
PLAYWRIGHT_PROJECT_K8S="showcase-k8s"
PLAYWRIGHT_PROJECT_RBAC_K8S="showcase-rbac-k8s"
PLAYWRIGHT_PROJECT_OPERATOR="showcase-operator"
PLAYWRIGHT_PROJECT_OPERATOR_RBAC="showcase-operator-rbac"
PLAYWRIGHT_PROJECT_RUNTIME="showcase-runtime"
PLAYWRIGHT_PROJECT_UPGRADE="showcase-upgrade"
```
Then use something like:
`check_and_test "${RELEASE_NAME}" "${NAME_SPACE}" "${PLAYWRIGHT_PROJECT_K8S}"`

Reply:
I considered doing this, but then I thought that it might be more readable to have it configured directly. 
Having too many global variables makes it less transparent 🤔 
But I don't have a strong opinion here 😄